storage.py

```diff
@@ -0,0 +1,25 @@
+from azure.mgmt.storage import StorageManagementClient
+from azure.mgmt.subscription import SubscriptionClient
+
+from auth import CREDENTIALS
+
+
+def list_subscriptions():
```

Comment:
I'm thinking we should have this as a separate file. Maybe merge it with authentication. We will end up reusing it everywhere.

Reply:
Approving still 👍